server/src/domain/example/post/post.routes.ts

```diff
@@ -0,0 +1,36 @@
+import { Type } from '@sinclair/typebox';
+
+import { type ServerInstance } from '~/setup/server';
+import * as postsService from '~/domain/example/post/post.service';
+
+export async function postRoutes(server: ServerInstance) {
```

Comment:
My idea was that this file is just a super simple example for how to define REST API endpoints. I was too lazy to add more endpoints to it 🙈 Stuff like this can probably be added later on by other people if we want to make the examples more elaborate.